### PR TITLE
Update references to study archives

### PR DESCRIPTION
**Pull request by labkey-adam**
#### Rationale
Primary goal is to modernize the descriptions and remove "study archive" references on the Folder Import page: [Issue 45014: Update descriptions in UI for study reload and folder import](https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=45014)

While we're at it, fix "study archive" references in other places in this repo to reduce confusion.

### CODE REVIEW

File: api/schemas/study.xsd [hunk 1]
```diff
@@ -5,8 +5,8 @@
            elementFormDefault="qualified" attributeFormDefault="unqualified">
 	<xs:annotation>
 		<xs:documentation xml:lang="en">
-            Describes a file-based study. This is the top-level file in a study archive. It defines the top-level study
-            settings and points to the other files in the archive.
+            Describes a file-based study. This is the top-level file in the study node of a folder archive. It defines
```
Comment (labkey-jeckels) [inline api/schemas/study.xsd]:
```suggestion
            Describes a file-based study. This is the top-level file in the study subdirectory of a folder archive. It defines
```
```suggestion
            Describes a file-based study. This is the top-level file in the study node of a folder archive. It defines
```

File: core/src/org/labkey/core/admin/importFolder.jsp [hunk 2]
```diff
@@ -61,41 +61,45 @@
...
-    String helpLinkTxt = "For more information about exporting and importing folders, see " +
-            helpLink("importExportFolder", "the folder documentation") + ".";
+    final String noun;
+    final String action;
+    final String mainDescription;
+    final String helpLinkTxt;
+
+    final String whatIsAFolderArchive = "A folder archive is a .folder.zip file or a collection of individual files that " +
+        "conforms to the LabKey folder archive conventions and formats. A folder archive can be created using the folder " +
+        "export feature or via scripts that write data from a master repository into the correct formats. ";
```
Comment (labkey-jeckels) [inline core/src/org/labkey/core/admin/importFolder.jsp]: Given our phasing out of "master"...

```suggestion
        "export feature or via scripts that write data from a master repository into the correct formats. ";
```suggestion
        "export feature or via scripts that write data from a source repository into the correct formats. ";

```
```

Comment (labkey-jeckels) [inline core/src/org/labkey/core/admin/importFolder.jsp]: Given our phasing out of "master"...

```suggestion
        "export feature or via scripts that write data from a master repository into the correct formats. ";
```suggestion
        "export feature or via scripts that write data from a source repository into the correct formats. ";

```
```

File: study/api-src/org/labkey/api/study/importer/SimpleStudyImporter.java [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 import org.labkey.api.admin.ImportException;
 
-// Implemented by importers that import from a study archive but are registered by other modules,
+// Implemented by importers that import from the study node of a folder archive but are registered by other modules,
```
Comment (labkey-jeckels) [inline study/api-src/org/labkey/api/study/importer/SimpleStudyImporter.java]:
```suggestion
// Implemented by importers that import from the study subdirectory of a folder archive but are registered by other modules,
```
```suggestion
// Implemented by importers that import from the study node of a folder archive but are registered by other modules,
```

Comment (labkey-jeckels) [inline study/api-src/org/labkey/api/study/importer/SimpleStudyImporter.java]:
```suggestion
// Implemented by importers that import from the study subdirectory of a folder archive but are registered by other modules,
```
```suggestion
// Implemented by importers that import from the study node of a folder archive but are registered by other modules,
```

File: study/src/org/labkey/study/writer/StudySerializationRegistry.java [hunk 4]
```diff
@@ -75,7 +75,7 @@ public static StudySerializationRegistry get()
         );
     }
 
-    // These writers are related to study and serialize into the study archive, but are registered by other modules
+    // These writers are related to study and serialize into the study node of a folder archive, but are registered by other modules
```
Comment (labkey-jeckels) [inline study/src/org/labkey/study/writer/StudySerializationRegistry.java]:
```suggestion
    // These writers are related to study and serialize into the study subdirectory of a folder archive, but are registered by other modules
```
```suggestion
    // These writers are related to study and serialize into the study node of a folder archive, but are registered by other modules
```

Comment (labkey-jeckels) [inline study/src/org/labkey/study/writer/StudySerializationRegistry.java]:
```suggestion
    // These writers are related to study and serialize into the study subdirectory of a folder archive, but are registered by other modules
```
```suggestion
    // These writers are related to study and serialize into the study node of a folder archive, but are registered by other modules
```

File: study/test/src/org/labkey/test/tests/study/StudyReloadColumnInferenceTest.java [hunk 5]
```diff
@@ -53,9 +54,9 @@
             -Study.dataset
         -study.xml
     * */
-    private static final File STUDY_ARCHIVE = TestFileUtils.getSampleData("studyreload/datasetsSetup.zip");
+    private static final File FOLDER_ARCHIVE = TestFileUtils.getSampleData("studyreload/datasetsSetup.zip");
 
-    /*Study archive:
+    /* Study node:
```
Comment (labkey-jeckels) [inline study/test/src/org/labkey/test/tests/study/StudyReloadColumnInferenceTest.java]:
```suggestion
    /* Study subdirectory:
```
```suggestion
    /* Study node:
```

Comment (labkey-jeckels) [inline study/test/src/org/labkey/test/tests/study/StudyReloadColumnInferenceTest.java]:
```suggestion
    /* Study subdirectory:
```
```suggestion
    /* Study node:
```

File: study/test/src/org/labkey/test/tests/study/StudyReloadColumnInferenceTest.java [hunk 6]
```diff
@@ -67,7 +68,7 @@
     * */
     private static final File initialReloadTestFile = TestFileUtils.getSampleData("studyreload/originalMixedFileTypes.zip");
 
-    /*Study archive:
+    /* Study node:
```
Comment (labkey-jeckels) [inline study/test/src/org/labkey/test/tests/study/StudyReloadColumnInferenceTest.java]:
```suggestion
    /* Study subdirectory:
```
```suggestion
    /* Study node:
```

Comment (labkey-jeckels) [inline study/test/src/org/labkey/test/tests/study/StudyReloadColumnInferenceTest.java]:
```suggestion
    /* Study subdirectory:
```
```suggestion
    /* Study node:
```

**Review by labkey-jeckels — APPROVED** [review]
Feel free to reject any of these suggestions. I can't think of other places we use "node" in this context

**Comment by labkey-adam** [comment]
@labkey-jeckels good suggestions. I went with "/study subfolder" for clarity and to cover the zipped archive case.

**Pull request merged**